awscli/botocore/utils.py

```diff
@@ -3305,6 +3339,231 @@ def fetch_token(
         )
 
 
+class SSOTokenFetcherAuth(BaseSSOTokenFetcher):
+    """Performs the authorization code grant with PKCE OAuth2.0 flow"""
+    _AUTH_GRANT_TYPES = ['authorization_code', 'refresh_token']
```

Comment:
If we don't intend for these to be mutated during runtime, it's generally better to declare them as a `tuple` for immutability.

i.e.
```suggestion
    _AUTH_GRANT_TYPES = ('authorization_code', 'refresh_token')
```